packages/marko-web-html-sitemap/templates/date-list.marko

```diff
@@ -46,8 +49,8 @@ $ switch (displayType) {
               <div class=`col__date col__date--${displayType}`>
                 <marko-web-link href=`${node.alias}` >
                   <if(displayType === 'month')>
-                    $ const d = new Date(`${node.year}/${node.month}`)
-                    ${moment(d).format("MMMM")}
+                    $ const d = dayjs(`${node.year}/${node.month}`);
```

Comment:
@Shinsina this should be $ const d = dayjs(`${node.year}/${node.month}`, 'YYYY/MM');` or `$ const d = dayjs(`${year}-${month}`, "YYYY-MM");` if we want to keep it the same as the breadcrumb stuff